### PR TITLE
Kendall 1-2

### PR DESCRIPTION
**Pull request by maijapapaya**
@jaredh159 @Henderjay

### CODE REVIEW

File: letters/modernized/02-letters.adoc [hunk 1]
```diff
@@ -1,7 +1,7 @@
-== Letters I &mdash; XVI
+== Letters &mdash; XVI
```
Comment (maijapapaya) [inline letters/modernized/02-letters.adoc:1]: I couldn't remember what I erased here when I did it, and then I wasn't even sure how you wanted this to look so I just left it and figured you could tell me here. :)

**Comment by jaredh159** [comment]
@Henderjay @maijapapaya -- whoops i messed something up here, but i'll fix it. sorry!

**Comment by jaredh159** [comment]
test comment

**Comment by jaredh159** [comment]
reopen!

**Pull request closed without merging**